app/models/v2-subscription.js

```diff
@@ -22,7 +22,7 @@ export default Model.extend({
   organization: belongsTo('organization'),
   coupon: attr('string'),
   clientSecret: attr('string'),
-  paymentIntent: attr('string'),
+  paymentIntent: attr(),
```

Comment:
Is this change intentional? What's the type of `paymentIntent`?

Reply:
This change is intentional, yes, it's an object